asv_bench/benchmarks/io/csv.py

```diff
@@ -69,13 +67,34 @@ def setup(self, test_filenames, shape, skiprows):
         self.skiprows = self.skiprows_mapping[skiprows] if skiprows else None
 
     def time_skiprows(self, test_filenames, shape, skiprows):
-        execute(
+        self.execute(
             IMPL[ASV_USE_IMPL].read_csv(
                 test_filenames[self.shape_id], skiprows=self.skiprows
             )
         )
```

Comment:
What if we go this way?
```suggestion
        execute(
            IMPL[ASV_USE_IMPL].read_csv(
                test_filenames[self.shape_id], skiprows=self.skiprows
            ),
            trigger_omnisci_import=True, 
        )
```